docs/tutorials/understanding_21cmsense.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<p>The formula for final averaged errorbars doesn't have the correct units. should it be 1/sqrt(sum(eppsilon^-2))?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/21cmSense/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yes.</p>